### DISCUSSION TITLE
restating question about ripgrep installers

### DISCUSSION THREAD

**Discussion by Phillip-M-Feldman** [BurntSushi/ripgrep#1647] [General] [answered]
I've read the instructions at https://github.com/BurntSushi/ripgrep#installation multiple times.

When I follow those instructions (go to https://github.com/BurntSushi/ripgrep/releases and start hitting Next), the third page says version 11.0.2 at the top, but the URL is `https://github.com/BurntSushi/ripgrep/releases?after=grep-matcher-0.1.4`.  This is confusing.  I don't see any file with the label 11.0.2, and when I unpack the 0.1.4 file, there is no installer.  So, I still haven't figured out.

**Comment by BurntSushi** [comment, marked as answer]
I don't understand what trouble you're running into or why you are looking for an old release, or why you are downloading a 0.1.4 release when you're looking for 11.0.2, but it's right here: https://github.com/BurntSushi/ripgrep/releases/tag/11.0.2

If you want a directly link to the x86_64 Windows MSVC zip archive, then here it is: https://github.com/BurntSushi/ripgrep/releases/download/11.0.2/ripgrep-11.0.2-x86_64-pc-windows-msvc.zip

**Comment by Phillip-M-Feldman** [comment]
Thanks so much!  It appears that there is a problem with at least one
release page: If one starts with the most recent release and hits Next
twice, one gets to a page that says 11.0.2 at the top but doesn't actually
have any files for that version.

**Reply by BurntSushi** [reply]
They are there. You just need to click on "Assets" and the files will expand.

**Comment by Phillip-M-Feldman** [comment]
Hm.  This is non-obvious.
